tv_apps/smart-home/bower_components/smart-button/style.css

```diff
@@ -446,3 +446,38 @@ smart-button[type="circle-text"].danger.pressed {
 /**
  * End of circle text button
  */
+
+/**
+ * rounded-corner text button
+ */
+smart-button[type="rounded-corner-text"] {
+  -moz-user-select: none;
+  width: auto;
+  height: 8rem;
+  border-radius: 4rem;
+  background-color: #5f6368;
+  padding: 0 3.8rem;
+  line-height: 8rem;
+  color: white;
+  text-align: center;
+  font-size: 2.4rem;
+  overflow: hidden;
+  transition-property: transform, color, background-color;
+}
+
+smart-button[type="rounded-corner-text"].focused,
+smart-button[type="rounded-corner-text"]:hover {
+  transform: scale(1.2);
+  color: #2d2d2d;
+  background-color: white;
+}
+
+smart-button[type="rounded-corner-text"].pressed {
+  transform: scale(1);
+  /*color: white;*/
```

Comment:
I just pulled the file from smart-button repository. Not sure why we have it :-/
Removing it should go from smart-button repository. Maybe opening another bug is suitable.